src/components/Form/SignInModal/SignIn/index.tsx

```diff
@@ -0,0 +1,89 @@
+import Input from '@components/Input'
+import Button from '@components/Button'
+import ErrorMessage from '@components/ErrorMessage'
+import { useFormik } from 'formik'
+import { signInSchema } from '@schemas/Signin'
+import { useAuth } from '@context/auth'
+import {
+  Wrapper,
+  Title,
+  Text,
+  ChangeFormBtn,
+} from '@components/Form/SignInModal/styles'
+
+type SignInProps = {
+  handleClick: (state: boolean) => void
+}
+
+export type SignInValues = {
+  email: string
+  password: string
+}
+
+const SignIn = ({ handleClick }: SignInProps) => {
+  const { signIn, isLoading, errorMsg } = useAuth()
+
+  const initialValues: SignInValues = {
+    email: '',
+    password: '',
+  }
+
+  const formik = useFormik({
+    initialValues: initialValues,
+    validationSchema: signInSchema,
+    onSubmit: (values: SignInValues) => handleSubmit(values),
+  })
+
+  const handleSubmit = (values: SignInValues) => {
+    const newSignInValues = {
+      user: {
+        ...values,
```

Comment:
Porque spread ao invés de só atribuir values para user?

Reply:
Realmente os dois dão na mesma, deixei passar batido. I will fix this.